wolox-books/src/app/components/auth/auth.component.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../../scss/colors.scss';
+@import '../../../scss//variables.scss';
+
+.navbar {
+  align-items: center;
+  background-color: $nav-background-color;
+  box-shadow: 7px 7px 6px $box-shadow-color;
+  display: flex;
+  height: 60px;
```

Comment:
use the navbar height as a constant in another file. It is very common to use this height to make calculations in other components